core-bundle/src/Command/SymlinksCommand.php

```diff
@@ -87,8 +87,13 @@ private function generateSymlinks(): void
         $this->symlinkModules();
         $this->symlinkThemes();
 
-        // Symlink the assets and themes directory
-        $this->symlink('assets', Path::join($this->webDir, 'assets'));
+        // Symlink the assets directory if the Contao components were installed there
+        // instead of in public/assets (backwards compatibility)
+        if (is_dir($this->projectDir.'/assets/contao')) {
```

Comment:
This will break if someone creates a `contao` subfolder in their `assets` directory.  We could check for multiple components instead, e.g. `assets/dropzone` and `assets/handorgel`, but that would also break if we remove one of them in the future.

@ausi Can you think of a better check that does not require reading the `composer.json` configuration?

Reply:
Check the other way round?
```suggestion
        if (!is_dir(Path::join($this->webDir, 'assets')) || is_link(Path::join($this->webDir, 'assets'))) {
```
Because if the components get installed to `public/assets` the directory would already be there at this point I guess?

But as this command is not performance relevant, I’d prefer reading the composer.json instead.

Reply:
Changed in cf4ddeda99e78dcbaeb9859b7b6e9a54fdba62ab.